modules/custom/falcon_commerce/falcon_commerce.module

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
Why do we create products here? As correctly pointed out by @PavelBulat - we need to create a "Donation" product only when donation is being enabled. So it makes sense to do it either in `falcon_feature_donation` or `falcon_donation` module.

Reply:
It is fixed. I chose falcon_donation module, because falcon_feature_donation automatically generated.